docs/content/docs/deep/container-and-component-loading.md

```diff
@@ -25,225 +25,295 @@ as well as the finished diagram in [Appendix 1](#appendix-1).
+The Operation Stream (op stream) is how Fluid stores state.
+State, including connected clients, the code to load, as well as distributed data structure modifications, are stored
+as a series of operations that when played in order produce the current state.
+I don't go into further details about it here.
 
 :::
 
 Connecting and processing the op stream includes:
 
-- Getting the Summary
-- Establishing the Websocket connection
-- Retrieving any missing ops from the REST endpoint
+- Getting the latest Summary of the container being loaded
+- Establishing the Websocket connection to the op stream
+- Retrieving any ops which are more recent than the latest Summary from the REST endpoint
+
+The Fluid Core (`Loader` + (Container) `Runtime`) is agnostic to how the Fluid Server is implemented.
+It instead uses a `Driver` model to allow for different servers to optimize for their own infrastructure.
 
 The `Driver` is responsible for taking the requests above **(3)** and transforming them to requests that the Fluid Server
 understands **(3.1)**.
 
-The Fluid Core (`Loader` + `Runtime`) is agnostic to how the Fluid Server is implemented. It instead uses
-a `Driver` model to allow for different servers to optimize for their own infrastructure.
-
-![Image 3](/images/container-and-component-loading-3.jpg)
+![Image 3](images/container-and-component-loading-3.jpg)
 
-The `Container` object itself does not actually do much. Once it has established a connection via the `Driver` its other
-responsibility is to listen specifically for one event emitted from the `Quorum`. This is the `"code"` proposal.
+The `Container` object itself does not actually do much.
+Once it has established a connection via the `Driver`, its other responsibility is to listen specifically for one event
+emitted from the `Quorum`.
+This is the `"code"` proposal, for the code that contains the `ContainerRuntime`.
 
 ::: tip
 
 The `Quorum` is a special key/value distributed data structure that requires all current members to agree on the value
-before the it is accepted. I don't go into further details about it here.
+before the it is accepted.
+I don't go into further details about it here.
+
+The `ContainerRuntime` is code written by the author of the Container, with all the logic that drives the functionality
+of the Container.
 
 :::
 
 There are a few different ways that the `Container` will get this `"code"` value:
 
-1. In the *Create New* flow this `"code"` value needs to be proposed by the Hosting Application. Once the value is
-   accepted by everyone connected (only you, the current client, in this case) the `Container` will get the event and
-   have the value.
-2. In the *Create from Existing* flow there are two scenarios.
+1. In the *Create New* flow this `"code"` value needs to be proposed by the Hosting Application.
+   Once the value is accepted by everyone connected (in this case only you, the current client) the `Container` will
+   get the event and have the value.
```

Comment:
Nit: this phrasing was hard for me to parse. Maybe something like the following would be better?
```suggestion
   get an event containing the agreed-upon value.
```